test/mosaic-integration-tests/src/test/java/org/eclipse/mosaic/test/app/mosaicandsumovehicles/SumoVehicle.java

```diff
@@ -60,6 +61,7 @@ public void processEvent(Event event) throws Exception {
     @Override
     public void onVehicleUpdated(@Nullable VehicleData previousVehicleData, @Nonnull VehicleData updatedVehicleData) {
         if (previousVehicleData == null) {
+            getLog().infoSimTime(this, "First update @{}", TIME.format(getOs().getSimulationTime()));
```

Comment:
same here

Reply:
I'm now capturing the `infoSimTime`-time with the regex.